packages/eslint-plugin/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
```

Comment:
Uncomment the heading, and add a subheading now there is changelog content for a new release

```suggestion
## Unreleased

### Changed
```